modules/new_profile/jsx/NewProfileIndex.js

```diff
@@ -194,7 +194,7 @@ class NewProfileIndex extends React.Component {
             name = "dateTakenConfirm"
             label = "Date of Birth Confirm"
             minYear = "2000"
-            maxYear = "2017"
+            maxYear = "2017-12-31"
```

Comment:
Should these values maybe not be hard coded? I'm not sure why there would be a boundary of 2000-2017 specifically for every study. I think this should be pulled from the `ageMin and `ageMax` config settings.

I know that's not really the scope of this PR so if you don't want to fix it just let me know and I'll open an issue for it.

Reply:
yes, you're definitely right!

Reply:
and no worries, i'll fix it